presto-product-tests/conf/docker/singlenode-kerberized/docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
     command: /docker/volumes/presto/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh singlenode-kerberized run
     volumes:
       - ../../../../:/docker/volumes/presto
-      - ../../../../presto-product-tests/conf/presto/etc/hive-kerberized.properties:/docker/volumes/presto/presto-product-tests/conf/presto/etc/catalog/hive.properties
+      - ${PRESTO_HIVE_CONFIG}:/docker/volumes/presto/presto-product-tests/conf/presto/etc/catalog/hive.properties
```

Comment:
@arhimondr : Since it's only a single volume change, @sopel39 and I decided to reuse the yml file. But, I am open with both the approaches. I will follow the same for simple_impersonation.